logbook-logstash/src/main/java/org/zalando/logbook/logstash/LogstashLogbackSink.java

```diff
@@ -17,22 +16,32 @@
 
 import static org.apiguardian.api.API.Status.EXPERIMENTAL;
 
-@AllArgsConstructor
```

Comment:
I'd leave that in, so you don't have to implement it by hand.

Reply:
Fine for me with your suggestion! Thanks